sdk/typescript/src/module/decorators/index.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * Expose the decorator publicly, so they insert data into the global registry.
  */
-import { registry } from "../registry/registry.js"
+import { registry } from "../registry/index.js"
```

Comment:
Because you cannot import from a directory `../registry`, it's not working with the configuration of our packages.

So I need to give the fulle path name with `index.js`

```
Relative import paths need explicit file extensions in ECMAScript imports when '--moduleResolution' is 'node16' or 'nodenext'. Consider adding an extension to the import path
```

And we need `node16` since it's the most recent resolution and some features depends on it (I tried to change it but it failed) with mocha.

Reply:
Note that I didn't say `../registry`, I said `../registry.js`. You already replied in another comment that you did this because you like to put classes in a directory, but it seems to me it adds unnecessary complexity. Unless it's a known convention.

Reply:
It's a classic convention yes, I did that for all my classes except directories that are composed of class only